Dockerfile

```
@@ -1,15 +1,17 @@
 FROM docker.io/python:3.8.6-slim-buster AS base
 
 ENV HOME=/app
+ENV PATH=${PATH}:${HOME}/.poetry/bin
```

You can do this in a single `ENV` directive (adding a backslash):
```suggestion
    PATH="${PATH}:${HOME}/.poetry/bin"
```

---

My bad, sorry. It would need to be `PATH="${PATH}:/app/.poetry/bin"` since `$HOME` is not set at that point.